nixos/doc/manual/man-nixos-version.xml

```diff
@@ -21,6 +21,10 @@
     <option>--revision</option>
    </arg>
 
+   <arg>
+    <option>--configurationRevision</option>
```

Comment:
as @SuperSandro2000 noted. I would changes this to `--configuration-revision` as this is more consistent with other nixos cli tools.